lib/elixir/lib/kernel.ex

```diff
@@ -3401,6 +3401,45 @@ defmodule Kernel do
   In the example above, a `sum/2` function is defined; this function receives
   two arguments and returns their sum.
 
+  ## Default arguments
+
+  `\\` is used to specify a default value for a parameter of a function. For
+  example:
+
+      defmodule Foo do
+        def mul_by(x, n \\ 2) do
```

Comment:
Also, let's use more descriptive function names and module names.

One example could be:

```
defmodule MyMath do
  def multiply_by(number, factor \\ 2) do
    number * factor
  end
end
```

or something like this.